pysat/tests/test_instrument.py

```diff
@@ -1209,17 +1209,15 @@ def test_right_bounds_with_next(self):
     def test_set_bounds_with_frequency(self):
         """Test setting bounds with non-default step"""
         start = self.ref_time
-        stop = self.ref_time + pds.DateOffset(days=14)
-        stop = stop.to_pydatetime()
+        stop = self.ref_time + dt.timedelta(days=14)
```

Comment:
This is the only bit I found where a specific pandas input was required, but since we're immediately converting to datetime we can just contract the code.